packages/global/scss/index.scss

```diff
@@ -709,6 +713,11 @@ body {
 .section-card-block {
   &__header {
     @include theme-card-header();
+    font-size: 32px;
+    font-weight: 800;
```

Comment:
800 is not a supported font weight for the font family being used.  [Oswald](https://fonts.google.com/specimen/Oswald) bold is 700 which is being inherited from the @include theme-card-header.  I would just remove the font-weight override.